### PR TITLE
Fix review page of incomplete add-ons for limited reviewers

### LINKED ISSUE #2969: Normal Reviewers cant access rejected addons after initial review
It seems that if an addon is reviewed and rejected for the first time (no prior history), normal reviewers cant access the review page. However, admin reviewers can.

Example:
`https://addons.mozilla.org/en-US/editors/review/socialmonkee`

> Oops! We had an error.
> 
> We'll get to fixing that soon.
> 
> You can try refreshing the page, or head back to the Add-ons homepage.

### PR DESCRIPTION
**Pull request by wagnerand**
Fix #2969.

### CODE REVIEW

File: src/olympia/editors/helpers.py [hunk 1]
```diff
@@ -557,6 +557,7 @@ def get_actions(self, request, addon):
             acl.action_allowed(request, 'ReviewerAdminTools', 'View'))
         reviewable_because_submission_time = (
             not is_limited_reviewer(request) or
+            not addon.latest_version or
             (datetime.datetime.now() - addon.latest_version.nomination >=
```
Comment (eviljeff) [inline src/olympia/editors/helpers.py]: what about when `latest_version is not None` but `latest_version.nomination `isn't set?

Comment (wagnerand) [inline src/olympia/editors/helpers.py]: That shouldn't happen (tm).

Comment (eviljeff) [inline src/olympia/editors/helpers.py]: like this: https://addons.mozilla.org/en-US/editors/review/max-access-management ?

**Comment by eviljeff** [comment]
Can you alter/add something so this changed is tested?

**Comment by wagnerand** [comment]
I'm not sure what a valid test-case would be. Normally, we shouldn't end up in this case, because if there is no latest version the add-on status would be `STATUS_NULL` which is covered a few lines above.

I don't know how an add-on would end up with a latest_version but no nomination date, though.

**Comment by eviljeff** [comment]
A test for when there isn't a `latest_version` and you're a limited reviewer is probably enough (the original issue).  I agree that having a latest version but no nomination is an edge case.

**Comment by wagnerand** [comment]
Test added. Turns out, it's not that easy to set `latest_version` manually.

**Comment by eviljeff** [comment]
r+

**Comment by wagnerand** [comment]
Thank you.

**Pull request merged**